vkconfig/dlgcreateassociation.cpp

```diff
@@ -37,7 +37,12 @@ dlgCreateAssociation::dlgCreateAssociation(QWidget *parent)
     Configurator &configurator = Configurator::Get();
     configurator._override_application_list_updated = false;
 
-    if (!configurator.HasActiveOverrideOnApplicationListOnly()) setWindowTitle("Applications Launcher Shortcuts");
+    if (!configurator.HasActiveOverrideOnApplicationListOnly())
```

Comment:
The two branches of the `if` aren't parallel at all.  I'd think that if `ui->treeWidget->setHeaderHidden(false)` had to be called in one branch, that `setHeaderHidden(true)` would have to be called in the other, etc.

Please double-check that this does what you think it does.  If you're convinced it does, consider a comment explaining why the two branches are not parallel.

Reply:
The default state is hidden, as set by the .ui file. I did look twice though, so it warrants a comment.